src/files-regular/refs-tests.js

```diff
@@ -0,0 +1,315 @@
+/* eslint-env mocha */
+'use strict'
+
+const async = require('async')
```

Comment:
Now that "async" is a language keyword this can get confusing.

```suggestion
const map = require('async/map')
```